### PR TITLE
Mobian installer added in v1.5

### PR DESCRIPTION
**Pull request by nuehm-arno**
This is the multiple times tested v1.5 of the installer script.

### CODE REVIEW

**Comment by sc0w** [comment]
your `axolotl.desktop` file is wrong for me

as I told you, I have other folders, for example:

I don't have `dist` folder inside `axolotl-web` folder, the `png` file is inside `axolotl-web` folder

I have `axolotl` folder inside `linux-arm64` folder with `axolotl` program

Here the path must be `/home/mobian/go/src/github.com/nanu-c/axolotl`, if not axolotl shows me white page with the error: 404 error

**Comment by nuehm-arno** [comment]
I cannot reproduce and since I helped a lot of people installing Axolotl with these commands, your case seems to be a rare exception, which I cannot explain.
Please re-install Axolotl with the script in version 1.5 - it should be fine.

**Comment by sc0w** [comment]
I don't know why we have different folders, anyway, for this reason I think the script must check if folders / files exists to decide which line to write on `axolotl.desktop` to fix this "rare exception" in your words

**Review by nanu-c — APPROVED** [review]

File: scripts/mobian_installer.sh [hunk 1]
```diff
@@ -1,28 +1,69 @@
...
-echo "[Desktop Entry]\nType=Application\nName=Axolotl\nGenericName=Signal Chat Client\nPath=/home/mobian/go/src/github.com/nanu-c/axolotl\nExec=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl/axolotl\n#Exec=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl/axolotl -e qt\nIcon=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl-web/axolotl.png\nTerminal=false\nCategories=Network;Chat;InstantMessaging;Qt;\nStartupWMClass=axolotl" | sudo tee -a /usr/share/applications/axolotl.desktop
+if [ -f /usr/share/applications/axolotl.desktop ];
+then
+  :
+else
+  echo "[Desktop Entry]\nType=Application\nName=Axolotl\nGenericName=Signal Chat Client\nPath=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/\nExec=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl\n#Exec=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl -e qt\nIcon=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl-web/dist/axolotl.png\nTerminal=false\nCategories=Network;Chat;InstantMessaging;Qt;\nStartupWMClass=axolotl" | sudo tee -a /usr/share/applications/axolotl.desktop
```
Comment (sc0w) [inline scripts/mobian_installer.sh:66]: echo without -e never writes the new lines \n

anyway the file axolotl.desktop in these new changes is wrong for me

@nanu-c did you test this PR?

Comment (nuehm-arno) [reply scripts/mobian_installer.sh:66]: `echo -e` is indeed not adding new lines via \n in a normal terminal window.
But a simple `echo` used in a sh-script is doing this. Just try and test it.
In the Mobian wiki, there was a copy&paste error in the commands for creating the desktop files, that is true and now fixed. Thanks for pointing out that in this case the `-e` was missing.

Regarding the other thing with your additional axolotl folder in linux-arm64:

In my opinion, the only command that could have caused the additional folder creation, could have been this

`env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl .`

and my question is: Did you used it exactly like that? With the dot at the end?

Comment (sc0w) [reply scripts/mobian_installer.sh:66]: yes, following the wiki step by step the .desktop file is wrong

probably it works with your script, I don't know, I didn't test it, but with the wiki the .desktop file doesn't work

did you test the wiki step by step? (without your script)

**Pull request merged**